src/main/webapp/app/quiz/shared/questions/short-answer-question/short-answer-question.component.ts

```diff
@@ -33,6 +33,7 @@ export class ShortAnswerQuestionComponent {
     @Input()
     set question(question: QuizQuestion) {
         this.shortAnswerQuestion = question as ShortAnswerQuestion;
+        this.showingSampleSolution = false;
```

Comment:
I think we can re-use the existing method here instead
```suggestion
        this.hideSampleSolution();
```